yarnitor/core.py

```diff
@@ -1,6 +1,6 @@
 from flask import Flask
-from flask.ext.redis import FlaskRedis
-from flask.ext.cache import Cache
+from flask_redis import FlaskRedis
+from flask_cache import Cache
```

Comment:
Are `flask_redis` and `flask_cache` new dependencies?

Reply:
flask.ext.* is deprecated. This silences warnings.